addons/website_slides/models/slide_channel.py

```diff
@@ -385,3 +416,13 @@ def _count_presentations(self):
             record.nbr_infographics = result[record.id].get('infographic', 0)
             record.nbr_webpages = result[record.id].get('webpage', 0)
             record.total_slides = record.nbr_presentations + record.nbr_documents + record.nbr_videos + record.nbr_infographics + record.nbr_webpages
+
+    @api.model
+    def reorder_categories(self, categories_data=None):
+        ids = [e['id'] for e in categories_data]
+        categories_to_change = self.browse(ids)
```

Comment:
Actually, since we are only writing on single record, we don't need to browse them all at once.